azurelinuxagent/ga/exthandlers.py

```diff
@@ -236,6 +236,10 @@ def run(self):
                       message=msg)
             return
 
+    def run_status(self):
+        self.report_ext_handlers_status()
+        return
```

Comment:
Unnecessary return

Reply:
I think I merged this wrong with the rebase, this should file should not have any changes.  Removed.

---
In reply to: [194024817](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194024817) [](ancestors = 194024817)